com.unity.multiplayer.mlapi/Runtime/Serialization/NetworkWriter.cs

```diff
@@ -18,17 +17,17 @@ namespace MLAPI.Serialization
     /// </summary>
     public class NetworkWriter
     {
-        private Stream sink;
-        private NetworkBuffer networkSink;
+        private Stream m_Source;
```

Comment:
Was this really a source before but mis-named, or might this be mis-named now?

Reply:
for `NetworkReader` it was `source` so I kept it consistent between these two (also source made more sense than "sink")

Reply:
I dunno, seems like we are just writing to this stream (and after all, the class is a writer), and a sink is the word I use for 'place where written things go' (opposite of the source), so I'd argue 'sink' is correct here, but source is correct in NetworkReader.

Reply:
sure, I'll convert these to `sink` variants.